rest/api.go

```diff
@@ -679,13 +694,14 @@ func sleep(rq *http.Request, d time.Duration) {
 func (h *handler) handleStats() error {
 	st := stats{}
 	runtime.ReadMemStats(&st.MemStats)
-
 	h.writeJSON(st)
+
+	base.Audit(h.ctx(), base.AuditIDSyncGatewayStats, base.AuditFields{"stats_format": "memstats"})
```

Comment:
I think this should have:

```
		mandatoryFieldGroups: []fieldGroup{
			fieldGroupRequest,
			fieldGroupAuthenticated,
		},
```

I also think (but did not research) that there is a behavior about how the auth information is stored on the context for a silent handler.